### ISSUE TITLE
aws package incorrectly parses "!GetAtt VPC.Outputs.VpcId"

### ISSUE THREAD

**Issue by romaninsh** [aws/aws-cli#2719]
As per documentation: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html , this is a valid code:

``` yml
    Parameters:
        VpcId: !GetAtt VPC.Outputs.VpcId
```

When this is parsed by `aws package` it is converted into:

``` yml
      Parameters:
        VpcId:
          Fn::GetAtt:
          - VPC
          - Outputs
          - VpcId
```

Which is invalid, because GetAtt supports only two arguments producing error:

"every Fn::GetAtt object requires two non-empty parameters, the resource name and the resource attribut"

The expected output is:

``` yml
      Parameters:
        VpcId:
          Fn::GetAtt:
          - VPC
          - Outputs.VpcId
```

**Comment by JordonPhillips** [comment]
Thanks for the report! Looks like it's bug. Looping in the service team to have a look.

@sanathkr

**Comment by sanathkr** [comment]
@romaninsh  This should have been fixed by https://github.com/aws/aws-cli/commit/9b69a93484f04ccef21686b0d7f7a21d0140e2e6. Can you update your CLI and test?

**Comment by stobrien89** [comment]
Closing, as fix was issued in [9b69a93](https://github.com/aws/aws-cli/commit/9b69a93484f04ccef21686b0d7f7a21d0140e2e6).
